inference-engine/src/transformations/src/transformations/common_optimizations/conv_bias_fusion.cpp

```diff
@@ -169,10 +175,13 @@ ngraph::pass::ConvAddFusion::ConvAddFusion() {
 NGRAPH_RTTI_DEFINITION(ngraph::pass::ConvMultiplyFusion, "ConvMultiplyFusion", 0);
 
 ngraph::pass::ConvMultiplyFusion::ConvMultiplyFusion() {
+    TRANSFORMATION_SCOPE(ConvMultiplyFusion);
     auto conv = ngraph::pattern::wrap_type<op::ConvolutionIE>(pattern::consumers_count(1));
     auto add = ngraph::pattern::wrap_type<opset1::Multiply>({conv, std::make_shared<pattern::op::Label>()});
 
-    matcher_pass_callback callback = get_callback<op::ConvolutionIE>();
+    matcher_pass_callback callback = get_callback<op::ConvolutionIE>([]() {
+        MATCHER_SCOPE(ConvMultiplyFusion);
```

Comment:
I think we don't need add logging in each callback. It is enough to do it in one place, where callback is called.

Reply:
This callback is called only if the callback was applied.

Reply:
@ilyachur I see. I mean that it is better to log callback call in one common place.

Reply:
I changed the convolution callback.